Assets/MixedRealityToolkit/_Core/Extensions/VectorExtensions.cs

```diff
@@ -161,5 +161,86 @@ public static bool IsValidPosition(this Vector3 position)
             return !float.IsNaN(position.x) && !float.IsNaN(position.y) && !float.IsNaN(position.z) &&
                    !float.IsInfinity(position.x) && !float.IsInfinity(position.y) && !float.IsInfinity(position.z);
         }
+
+        /// <summary>
+        /// Get the relative mapping based on a source Vec3 and a radius for spherical mapping.
+        /// </summary>
+        /// <param name="source">The source <see cref="Vector3"/> to be mapped to sphere</param>
+        /// <param name="radius">This is a <see cref="float"/> for the radius of the sphere</param>
+        /// <returns></returns>
+        public static Vector3 SphericalMapping(Vector3 source, float radius)
+        {
+            float circ = 2f * Mathf.PI * radius;
+
+            Vector3 newPos = new Vector3(0f, 0f, radius);
```

Comment:
why make a new vector here? couldn't we just assign it back to the source?

Reply:
It looks like some of Kurtis' changes got lost in the port, you can see this was fixed in the RadialMapping() function in ObjectCollection.cs here:
https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/2246/files/64f46d85b93b59faf2fd93f69a18deb9f706cdb0